<launch>
  <group ns="topside">
  <rosparam command="load" file="$(find cart2)/data/config/$(env LOCATION).yaml" />  	
	<node pkg="joy" name="joystick" type="joy_node">
		<param name="dev" value="$(env JOYSTICK)" />
		<param name="coalesce_interval" value="0.1" />
		<param name="autorepeat_rate" value="10" />
		<param name="deadzone" value="0.05" />
	</node>

	<node pkg="cart2" type="topside_radio" name="topside_modem">
		<!-- <param name="PortName" value="/dev/ttyUSB1" /> -->
		<param name="PortName" value="/dev/pts/8" />
		<param name="BaudRate" value="19200" />
		<param name="TransmitRate" value="3" />
		<param name="tf_prefix" value="topside" />
		<remap from="joy_in" to="joy" />
		<remap from="target_point" to="/dummy/stateHat" />
	</node>

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="topside_reconfigure" />
	<node pkg="rviz" name="rviz_topside" type="rviz" />
  </group>
</launch>
